mergify_engine/actions/merge/queue.py

```diff
@@ -116,8 +116,14 @@ def _move_pull_to_new_base_branch(self, pull_number, old_base_branch):
             gh_pull=pull_number,
         )
 
+    def is_first_pull(self, pull_number):
+        pull_requests = self.get_pulls()
+        if not pull_requests:
+            return True
+        return pull_requests[0] == pull_number
+
     def get_pulls(self):
-        return self.redis.zrange(self._cache_key, 0, -1)
+        return [int(pull) for pull in self.redis.zrange(self._cache_key, 0, -1)]
```

Comment:
why is that required now?

Reply:
I just moved where it happens, Set keys has always been a string. Before this was done just after we were calling `get_pulls()`.

Reply:
Makes sense!